_emails/louisville.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Louisville, KY
+permalink: "/lousiville-justice"
+name: Letter to Mayor - Justice for Breonna Taylor
+state: KY
+city: Louisville
+layout: email
+recipients:
+- greg.fischer@louisvilleky.gov
+- attorney.general@ag.ky.gov
+subject: I Want Justice  for the Murder of Breonna Taylor
+body: |
+  Dear Mayor Fischer,
+  
+  My name is [insert name]. I am a resident of [insert city & state] and I am emailing today to demand accountability for the racist and brutal murder of Breonna Taylor.
+  
+  I demand that charges be pressed against all officers involved in this heinous crime and its coverup, especially John Mattingly, Brett Hankinson, and Myles Cosgrove. There have been no charges thus far and this is unacceptable.
+  
+  Breonna Taylor would be alive today if it was not for the gross abuse of power and negligence exhibited by the Louisville Metro Police Department; all officers involved must face consequences for this murder in order to provide her family with justice and prevent further cops from committing brutal acts of violence against our communities.
+  
+  In addition, I demand that you start providing more support towards community efforts and organizations outside of the police forces in order to prevent police brutality and violence in the future.
+  
+  Sincerely,
+  [insert name]
```

Comment:
We should include name, phone number, email, and address here

also, would you  mind making all of the text inside the brackets capitalized? e.g., [NAME][EMAIL] etc.

This way people will be sure not to miss it :^)

Reply:
Done!